packages/esm-patient-registration-app/src/patient-registration/section/death-info/death-info-section.test.tsx

```diff
@@ -15,16 +16,18 @@ jest.mock('@openmrs/esm-framework', () => {
 });
 
 // TODO: Implement feature and get tests to pass
-describe.skip('death info section', () => {
+describe('death info section', () => {
   const formValues: FormValues = initialFormValues;
 
   const setupSection = async (isDead?: boolean) => {
     render(
-      <Formik initialValues={{ ...initialFormValues, isDead }} onSubmit={null}>
-        <Form>
-          <DeathInfoSection />
-        </Form>
-      </Formik>,
+      <PatientRegistrationContext.Provider value={{ values: { isDead: isDead || false } }}>
```

Comment:
isDead || false can be replaced with !!dead

Reply:
isDead is not meant for `PatientRegistrationContext.Provider`.

Reply:
> isDead is not meant for `PatientRegistrationContext.Provider`.

with !!isDead. right?

Reply:
> isDead is not meant for `PatientRegistrationContext.Provider`.

dead-info-section is not rendered in the ui and the death info is not collected from the registration form but in the code there is one component for the death-info-section and also one story open for this feature https://issues.openmrs.org/browse/O3-303. So according to the story if the isDead will be checked then only this section will appear. But currently we do not have any implementation on this but isDead is defined on the FormValues types [here is the link](https://github.com/openmrs/openmrs-esm-patient-management/blob/4f28ea5189c48f22a81eb7b2f94d2efadd336b64/packages/esm-patient-registration-app/src/patient-registration/patient-registration-types.tsx#L172). 

That's why I used it here. If something need to be changed, Please let me know.

Reply:
> with !!isDead. right?

Yes @ayush-AI